components/theme/sizes.tsx

```diff
@@ -5,5 +5,5 @@ export const fontSizes = {
   sm: '0.85rem',
   md: '1rem',
   lg: '1.25rem',
-  xl: '1.5rem'
```

Comment:
I think it would've been better to create another one called `xxl: 3rem` for example.